sdk/storage/tests.yml

```diff
@@ -1,30 +1,24 @@
 trigger: none
 
 extends:
-  template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
+  template: ../../eng/pipelines/templates/stages/archetype-sdk-tests.yml
   parameters:
     ServiceDirectory: storage
     BuildInParallel: true
     TimeoutInMinutes: 180
-    Location: canadacentral
     CloudConfig:
       Preview:
-        SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources-preview)
-      PrivatePreview:
+        SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources-preview-test)
+        Location: 'canadacentral'
+      UsGov:
+        SubscriptionConfiguration: $(sub-config-gov-test-resources-test)
+      China:
+        SubscriptionConfiguration: $(sub-config-cn-test-resources-test)
+      PrivatePreview:
         SubscriptionConfiguration: $(sub-config-storage-test-resources)
-    Clouds: Preview
+    Clouds: 'Preview,UsGov,China'
     SupportedClouds: Preview, PrivatePreview
-    MatrixReplace:
-      # Use dedicated storage pool in canadacentral with higher memory capacity
-      - Pool=(.*)-general/$1-storage
```

Comment:
All the code removed here needs to stay (not sure whether you removed it for testing or intended for it to make it to the PR). A few reasons:

1. Storage has a custom agent pool with higher memory VMs, which is why we do this `MatrixReplace` to switch the pool.
2. Storage needs to test all service versions, but we've decided in the past to keep the cross-version testing only on the weekly builds.

Are there functional issues with either of the above configurations that you are seeing when testing sovereign clouds? If so, perhaps I can help?